openedx/features/enterprise_support/utils.py

```diff
@@ -401,7 +402,15 @@ def is_enterprise_learner(user):
     Returns:
         (bool): True if given user is an enterprise learner.
     """
-    return EnterpriseCustomerUser.objects.filter(user_id=user.id).exists()
+    cached_is_enterprise_key = '{}:user_is_enterprise'.format(user.id)
```

Comment:
@alex-sheehan-edx, at the risk of me being a newbie, is there a reason you chose not to use the get_cache_key utility that hashes your key?  The utility is used to hash the key and in many other places and I suspect it is for speed improvements.

Reply:
did some looking into `edx-django-utils` to see what the underlying `get_cache_key` is doing. Seems like we're just taking a close equivalent to the string here and converting it to an md5 hash. Talked offline and I agree/am sold on using it here for uniform key size and evenly spreading out keys within the cache data structure (although I'm unfamiliar with the specifics on django's caching system so I don't completely understand what that structure is).